google/resource_storage_bucket_acl.go

```diff
@@ -215,7 +237,11 @@ func resourceStorageBucketAclUpdate(d *schema.ResourceData, meta interface{}) er
 			}
 		}
 
-		for entity, _ := range old_re_map {
+		for entity, role := range old_re_map {
+			if entity == "project-owners-"+strconv.FormatInt(p.ProjectNumber, 10) && role == "OWNER" {
```

Comment:
Would it be easier to read to do `fmt.Sprintf("project-owners-%d", p.ProjectNumber)` instead of string concatenating `strconv.FormatInt`?

Reply:
Fair, I'll do that. Sorry, I'm just used to the non-`reflect`-based string construction methods from an old performance-sensitive project, and now they're reflex. Thanks for catching it :)